pyproject.toml

```diff
@@ -18,3 +18,21 @@ exclude = '''
 # make it compatible with black
 profile = "black"
 line_length = 120 # If this is removed, isort fights with Black
+
+
+# Linting tools configuration
+[tool.flake8]
```

Comment:
Please change it to `[tool.pflake8]`.

Reply:
Otherwise `pflake8` won't pick up the configuration.

Reply:
I did a quick github search and google search, I don't think [tool.pflake8] is a valid thing.
What I did was that in the pflake8 run string, I pass the --config which seemed to pick up the new configuration.